migrations/20170508160924_app_ldu.js

```diff
@@ -0,0 +1,16 @@
+
+exports.up = function(knex, Promise) {
+  return knex.schema.createTable('ldu', function (table) {
+    table.increments('id')
+    table.string('description')
+    table.timestamp('effective_from').defaultTo(knex.fn.now())

```

Comment:
From discussions earlier - we should probably create a region table which is basically just `(id, description)` and then have that be referenced in this table.

Each LDU will have one parent region.